benchmarking/evaluations/metrics/average_social_satisfied.py

```diff
@@ -16,3 +17,6 @@ def calculate(self, team_set: TeamSet) -> float:
             sum([self.metric_function(team) for team in team_set.teams])
             / team_set.num_teams
         )
+
+    def stdev(self, team_set: TeamSet) -> float:
```

Comment:
nit;

we kinda have been calling this `calculate_stdev` in other metric classes

tbh we should probably just add this to the abstract class since we're using it on so many metrics, I don't want it to be called like 7 things from class to class (this part can be a separate issue/PR)